Doppler.Currency/Dtos/CurrencyDto.cs

```diff
@@ -1,16 +1,20 @@
 using Newtonsoft.Json;
+using System.ComponentModel;
+using System.Runtime.Serialization;
 
 namespace Doppler.Currency.Dtos
 {
     public class CurrencyDto
     {
         public string Date { get; set; }
-
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
         public decimal SaleValue { get; set; }
-
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public decimal? BuyValue { get; set; }
         public string CurrencyName { get; set; }
         public string CurrencyCode { get; set; }
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
En este caso no lo pondría, lo dejaría siempre visible

Reply:
Lo puse por este comentario: "hiding this property in the JSON when it is false would be nice.". Pero lo puedo sacar

Reply:
updated

Reply:
![image](https://user-images.githubusercontent.com/70591946/93129319-2ccfab00-f6a7-11ea-85c0-47359dcb9040.png)